src/gsi/setupw.f90

```diff
@@ -1880,7 +1880,6 @@ subroutine contents_netcdf_diag_(udiag,vdiag)
               call nc_diag_data2d("v_Observation_Operator_Jacobian_val", real(dhx_dx_v%val,r_single))
            endif
 
-           call nc_diag_data2d("atmosphere_pressure_coordinate", exp(prsltmp)*r1000)
```

Comment:
Lines 1836-1837 and 1842-1843 contain u/v forecast_adjusted and forecast_unadjusted lines which can be removed.

https://github.com/NOAA-EMC/GSI/blob/40e27c19b62ecc84d9becc1d4558fc5baab06cfb/src/gsi/setupw.f90#L1836-L1837

https://github.com/NOAA-EMC/GSI/blob/40e27c19b62ecc84d9becc1d4558fc5baab06cfb/src/gsi/setupw.f90#L1842-L1843

Reply:
@CoryMartin-NOAA.  Do regional developers want or need to retain these two lines?   I don't know.    The regional lines were added in PR #309.  The same PR added these lines to the global block.   @emilyhcliu Emily, has Ming replied to your inquiry regarding the origin of these lines?

Reply:
> @CoryMartin-NOAA. Do regional developers want or need to retain these two lines? I don't know. The regional lines were added in PR #309. The same PR added these lines to the global block. @emilyhcliu Emily, has Ming replied to your inquiry regarding the origin of these lines?

No, Ming has not replied to me yet.

Reply:
OK.  Since the code in question is not executed in global runs we can revisit this when it's time to merge v16.3 changes back into develop.